packages/vue/src/components/organisms/SfFooter/SfFooter.stories.scss

```diff
@@ -3,13 +3,77 @@
   &__social{
     display: flex;
     justify-content: space-between;
-    padding: var(--spacer-base) var(--spacer-xl);
+    align-items: flex-start;
```

Comment:
In general, please do not use story-specific css for component since it may give user the wrong impression on the component layout. Please remove this file completely